randovania/games/fusion/logic_database/Sector 2 TRO.txt

```diff
@@ -560,7 +560,7 @@ Extra - room_id: [18]
   * L0 Hatch to Zazabi Speedway/Door to Zazabi Arena
   * Extra - door_idx: (133,)
   > Door to Zazabi Access
-      Impossible
+      Screw Attack and Speed Booster and After Boss Zazabi Defeated and Shinespark Tricks (Expert) and Disabled Door Lock Rando and Disabled Entrance Rando
```

Comment:
The more i think about it, the more i think itd be better to just keep this at impossible. 
You're currently missing also lv0 for that connection, as well as HJ (because i dont think you can spark with only WJ or SJ). And these are so many requirements for such a niche thing that is never in logic, that we might as well remove it.

Reply:
Isn't lv0 already being accounted for by the door connection itself? From what I remember all the doors have an indicator on their security level that is already being taken into account. Do we have a video of this being possible (I am assuming this is from the left to the right to get to the speedway.)? Just thinking about it, having to carry a shinecharge through 2 rooms and sparking in the third room just doesn't sound possible based on the timing alone.

Reply:
> that is already being taken into account

it usually is, in this case it isn't. Because logic looks at "can i reach the left door from the right door". if you tell it "only on non entrance rando and non door lock rando", then logic will expect you that if you spawn right at the door without lv0, that you are able to make it which you cant.